### DISCUSSION TITLE
AI輔助篩檢

### DISCUSSION THREAD

**Discussion by Zoe06060606** [1101-GS4719-C/Python-Community-GS4719-C#713] [General]
大腸癌為國病之一。透過AI輔助篩檢提高了大腸鏡瘜肉偵測率，此AI模型準確率高達95%，此模型偵測到瘜肉時，會提醒醫生進行判讀，這項技術能減少醫生判讀的錯誤率以及使人力資源能更精簡有效地利用。同時也增加技術維護的人員的職缺。
我認為這樣的跨域合作十分有利於醫療的進步，雖然同時可能會出現判讀錯誤導致醫療疏失的部分，並產生法律相關的問題，責任應該歸屬於醫生還是人工智慧的公司，法律方面仍須進行討論。

https://www.bnext.com.tw/article/71856/aetherai-endo

**Comment by Chiway1229** [comment]
大腸癌身為台灣最常見癌症之一，有這項技術後想必能對醫學健康上有著顯著進步
對於國人健康也是更進一步的進展呢!

**Comment by danwan1027** [comment]
透過AI輔助能讓準確度上升，對大家的健康福祉是一大進展！

**Comment by Yunhans** [comment]
之後看病就靠AI了

**Comment by AnAnistired** [comment]
AI真的是人類未來文明發展必要的元素，分享一下我的理想國：無政府無國界，讓一個超級AI統治世界解決問題，人有不定性、不公正的問題，讓超級AI當基準，人類就可以不用再浪費時間、資源去處理國際、政治問題，使資源損耗降到最低，延續人類文明。

**Comment by ghost** [comment]
這類的AI輔助判斷或許真的是未來趨勢，但這類的AI都需要很多數據去訓練，可現在礙於法規的病人隱私，這些要訓練的數據在獲取上都很麻煩，說不定未來除了捐血捐器官，還能捐這類能幫ai訓練的病例照片和數據呢～

**Comment by manipulateed** [comment]
AI確實增進整體人類福祉，透過不斷創新的運用，未來的各行各業AI將無所不在。

**Comment by williamyen0623** [comment]
我朋友曾經夢到自己得到大腸癌，似乎在台灣是個不容小覷的疾病，慶幸的是，科技漸漸能幫助人類醫療技術的進步

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
AI真的讓我們的生活越來越方便

**Comment by ru0372572** [comment]
醫療器具隨著時代漸漸進步，然而也有很多人付不起這些器材，因為新的東西一開始都會先很貴，再慢慢降下來。在這個議題中，我認為商業運作模式及成本也是值得討論的地方。

**Comment by Jasper525** [comment]
AI的醫療用途看似完美，但還會有倫理方面考量。舉例來說，因為AI準確率不可能100%，當辨識錯誤而產生醫療糾紛時誰應該要負責？因此，人工智慧的社會應用，有許多需要我們深思的細節。

**Comment by moonnight3503** [comment]
AI的應用著實廣泛，能應用在醫療方面真的是太優質了。

**Comment by ABCDEFGHIJKLMNO12345678** [comment]
相信未來社會 AI可說是無所不在 上至科技軟體到民生用品都能搭上邊

**Comment by Pietra1226** [comment]
AI著手在醫療上真的是對人類的一大福音，雖然有著倫理的諸多問題，但目前這些應用AI的面向來看，絕大部分是利大於弊的。

**Comment by a45320614** [comment]
科技進步的動力由戰爭轉變為醫療，對人類而言應該是一件好事吧

**Comment by ruthchen1020** [comment]
希望台灣可以多發展這個，感覺未來會很需要這樣的醫療輔具。
才能維持台灣在醫療上的地位。

**Comment by swerfg03joe** [comment]
AI用在醫療領域是很好的想法，除了可以快速精準地找出病灶，還能把初期正在病變的組織也一並標示出來，對醫療健康的發展作出很大的貢獻，因早期治療與預防可以省下相當多的時間與金錢
